src/main/java/lotto/model/WinningMoney.java

```diff
@@ -0,0 +1,46 @@
+package lotto.model;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum WinningMoney {
+	OTHER(0, 0), 
+	THREE(3, 5000), 
+	FOUR(4, 50000), 
+	FIVE(5, 1500000), 
+	SIX(6, 2000000000);
```

Comment:
다음과 같이 자릿수에 `_`를 사용하면 가독성을 높일 수 있어요.
```suggestion
	OTHER(0, 0), 
	THREE(3, 5_000), 
	FOUR(4, 50_000), 
	FIVE(5, 1_500_000), 
	SIX(6, 2_000_000_000);
```